docs/demand/transportation_electrification/methodology.rst

```diff
@@ -0,0 +1,761 @@
+:cite:p:`NREL_efs`. NREL provides a range of possible BEV fuel economies (“Slow
+Advancement”, “Moderate Advancement”, and “Rapid Advancement”). The charging model uses
+mid-range values as the default fuel economy for each vehicle category and year, as
+shown in :numref:`bev_fuel_economy_table`. 
+
+.. _bev_fuel_economy_table:
+
+.. table:: Default BEV fuel economy by vehicle category and year
+
+    +-------------------------------------+---------------------------+
+    |                                     | Fuel Economy (mile/GGE)   |
+    +                                     +------+------+------+------+
+    | Vehicle Type                        | 2017 | 2030 | 2040 | 2050 |
+    +=====================================+======+======+======+======+
+    | LDV BEV cars, 100 mile range        | 138  | 156  | 158  | 160  |
+    +-------------------------------------+------+------+------+------+
+    | LDV BEV cars, 200 mile range        | 129  | 152  | 155  | 156  |
+    +-------------------------------------+------+------+------+------+
+    | LDV BEV cars, 300 mile range        | 122  | 148  | 152  | 154  |
+    +-------------------------------------+------+------+------+------+
+    | LDT BEV trucks, 100 mile range      | 103  | 105  | 106  | 107  |
+    +-------------------------------------+------+------+------+------+
+    | LDT BEV trucks, 200 mile range      | 97   | 103  | 104  | 104  |
+    +-------------------------------------+------+------+------+------+
+    | LDT BEV trucks, 300 mile range      | 94   | 100  | 101  | 102  |
+    +-------------------------------------+------+------+------+------+
+    | Transit buses                       | 13   | 16   | 18   | 19   |
+    +-------------------------------------+------+------+------+------+
+    | MDV trucks                          | 16   | 21   | 23   | 24   |
+    +-------------------------------------+------+------+------+------+
+    | HDV trucks                          | 12   | 16   | 17   | 18   |
+    +-------------------------------------+------+------+------+------+
+
+Smart Charging Optimization Algorithm 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The smart charging algorithm was developed by :cite:p:`2014:zhang`. The algorithm is a
+least cost optimization problem, with the cost function as follows:
+
+.. math::
+
+    min \sum_{i=1}^n \sum_{j=1}^{seg(i)} f_{ij} x_{ij}
+
+| where:
+| :math:`f` is the cost of electricity in [$/kWh],
+| :math:`x` is the increase in vehicle state of charge in [kWh],
+| :math:`i` is the dwell period per trip count,
+| :math:`seg(i)` is the number of dwell segments for dwell period :math:`i`,
+| :math:`j` is the dwell segment in [h],
+| :math:`n` is the total number of dwell per trip periods
+
+The total number of dwell segments :math:`seg(i)` will depend on the how long the
+vehicle is parked.
+
+**Equality** constraint:
+
+.. math::
+
+    \sum_{i=1}^n \sum_{j=1}^{seg(i)} x_{ij} + \sum_{i=1}^n y_i
```

Comment:
Equal sign is missing in this equality constraint. Should `+` be `=` instead?

Reply:
Good catch. It should be equal to 0